lia-demo/src/main/java/com/lithium/community/android/example/LauncherActivity.java

```diff
@@ -0,0 +1,7 @@
+package com.lithium.community.android.example;
+
+import com.lithium.community.android.ui.components.activities.LiSupportHomeActivity;
+
+public class LauncherActivity extends LiSupportHomeActivity {
+//DO NOTHING
```

Comment:
Not quite understanding what this is for? Is it that the exposed activity was LauncherActivity and we are keeping it compatible with that? Ideally we should not extend if don't have anything to add/enhance

Reply:
LiSupportHomeActivity is already registered in lia-ui module, but is not launcher there.
in example application ideally - clients are supposed to implement there on launching points/entry points to community. This is just a proxy entry from demo app's Manifest file as a launcher activity.